security/fma-supervisor.md

```diff
@@ -0,0 +1,269 @@
+
+# FMs:
+
+There are three conceptual ways for the Supervisor to fail:
+
+- It is unresponsive
+- It is incorrect
+- It is destructive to the Nodes connected to it in some other way
+
+And all Failure Modes are some subtype. Incorrect responses are much worse than no responses, as they may mislead the network. At maximum, a network which is mislead may have fraudulent interop messages played on it, resulting in arbitrary damage to that network. I say “mislead” because all invalid interop messages are invalid by the protocol, and this fact can be checked using L1 data. But because there is limited client diversity, bugs may have greater impact.
+
+## FM1a: Supervisor Is Totally Unavailable
+
+- Description
+    - The Supervisor is entirely unavailable, as if it were disconnected.
+    - No Message/Block Validity could be determined for any Node connected to this Supervisor.
+    - Any Node in Managed Mode would be unable to advance their chain state at all besides local-unsafe.
+    - Any Node in Standard Mode would be unable to advance cross-unsafe and cross-safe, but could still advance local-unsafe and local-safe. And because Validity can’t be checked, the Node may prefer not to trust Safe data either.
+- Risk Assessment
+    - Low Impact, High likelihood.
+    - Software fails; ports become blocked. At some point an interruption will take down a Supervisor.
+    - We have redundancy solutions being designed [here](https://github.com/ethereum-optimism/design-docs/pull/218/files?short_path=88594e4#diff-88594e47f0a70261441a7452448ef1f240c7c0f15b9132c7789b2ee2d0e07bd2), which will make Supervisor failures less impactful for Chain Operators.
+    - Other Node Operators can adopt similar redundancy measures, but may follow their own infrastructure designs . Operators should treat the Supervisor as consensus critical, and manage it similar to how they would manage their L1 source.
+
+## FM1b: Supervisor Is Corrupted
+
+- Description
+    - Not only is the supervisor unavailable, its database has been corrupted or destroyed.
+    - In this scenario, all of the above applies, and the recovery is delayed by our ability to restore a working Supervisor.
+- Risk Assessment
+    - Low Impact, Low likelihood.
+    - The Supervisor has robust database trimming to clear invalid states and partial writes.
+    - This code, like all Supervisor code, could be tested more.
+- Mitigations
+    - There is a feature of the Supervisor that databases can be replicated over HTTP. If a Supervisor needs to be quickly synced it can bootstrap its databases from an existing Supervisor.
+    - Supervisors themselves should be made redundant, so if one goes down, another may serve in its place. Supervisor *replacement* is not something that is well tested, and more likely you’d switch to use that backup Supervisor *and* its Managed Nodes.
+
+## FM2a: Supervisor Incorrectly Determines Message Validity in Response to Sequencer - Unsafe Block Only
```

Comment:
Action item: communicate clearly to applications that depend on unsafe blocks the new risks that come with interop. Across is an example application